resources/sass/app.scss

```diff
@@ -337,14 +336,14 @@ dl.import-meta .download-csv {
             transition-property: background-color, border-color;
         }
         &:active {
-            background-color: $color-utility-red-30;
-            border-color: $color-utility-red-30;
+            background-color: $wikit-Button-primary-destructive-active-background-color;
+            border-color: $wikit-Button-primary-destructive-active-border-color;
             transition-property: background-color, border-color;
         }
         &:focus {
-            box-shadow: $color-base-100 0 0 0 1px inset;
-            border-color: $color-utility-red-50;
-            background-color: $color-utility-red-50;
+            box-shadow: $color-base-100 0 0 0 1px inset; // TODO not $wikit-Button-primary-destructive-focus-box-shadow?
```

Comment:
The token exists: https://github.com/wmde/wikit/blob/master/tokens/properties/components/Button.json#L323

Reply:
Done in bba763be328c5e94d3eda0be8acab4717fca42cc.